R/plots.R

```diff
@@ -348,6 +354,7 @@ transformDates <- function(plotsols){
   # Convert to POSIXct if feasible. This will require some thought to figure out
   # which sorts of Period formats can be generated by predLoad and predConc. In
   # the meantime, here we address just a couple of the possibilities.
+  is.POSIXt=DATE=is.Date=NULL
```

Comment:
This is fixed here https://github.com/miguelcleon/loadflex/blob/master/R/plots.R#L350 
and here
https://github.com/miguelcleon/loadflex/blob/master/R/plots.R#L354